src/frontend/components/Header/Header.js

```diff
@@ -0,0 +1,37 @@
+import "./Header.css";
+import {Link} from "react-router-dom";
+import { useState } from "react";
+const Header=()=>{
+  const[resnav,setResnav]=useState(false)
+
```

Comment:
The naming here can be a bit more meaningful. `resnav` doesn't convey anything specific.